airflow/config_templates/default_airflow.cfg

```diff
@@ -432,7 +432,8 @@ scheduler_heartbeat_sec = 5
 # -1 indicates to run continuously (see also num_runs)
 run_duration = -1
 
-# after how much time (seconds) a new DAGs should be picked up from the filesystem
+# After how much time (seconds) a new DAGs should be picked up from the filesystem.
+# Note that this is also increases interval between new tasks being scheduled
```

Comment:
nit:  `Note that this also increases the interval between new tasks being scheduled`

And could you help to elaborate on why it is the case?